src/ensembl/src/shared/components/help-popup/useHelpPopupHistory.ts

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
I guess this file can also be created when it is actually needed.

Reply:
✅